packages/core/database/lib/entity-manager/index.js

```diff
@@ -268,9 +268,12 @@ const createEntityManager = (db) => {
         throw new Error('Nothing to insert');
       }
 
-      await this.createQueryBuilder(uid).insert(dataToInsert).execute();
+      const createdEntries = await this.createQueryBuilder(uid).insert(dataToInsert).execute();
 
-      const result = { count: data.length };
+      const result = {
+        count: data.length,
+        ids: createdEntries.map((entry) => (typeof entry === 'object' ? entry?.id : entry)),
```

Comment:
minor comment, we just discovered today `typeof null === 'object'` is true in javascript , it should not a problem here though

Reply:
Haha yes this one I know about :) that's you we use isNill in a lot of places ;)